wikimedia/wikipedia-ios
# Batch 1 - Change summary tags to edit tags

**Phabricator:** https://phabricator.wikimedia.org/T366730

### Notes
This PR updates our API calls for undo, rollback, and adding / changing an article description to use edit tags instead of appending hashtags to the edit summary.

### Test Steps
_Note: this isn't exhaustive, I suggest we lean on QA more once it's deployed to other Wikis_
1. Go to an article with Test Wiki that _already_ has an article description.
2. Change the article description and publish.
3. Go to article history on Desktop, confirm edit summary no longer has `#article-change-desc` appended to the edit summary, and instead has the `App description change` tag added.
4. Go to a random article history on Test Wiki, go into a diff. Tap Undo.
5. Go to article history on Desktop, confirm edit summary no longer has `#diff-undo` appended to the edit summary, and instead has the `App undo` tag added.
